pagopa/pn-frontend
# feat/PN-11563 - Add 'download jpeg' button to sender dashboard

## Short description
This code adds the Screenshot module to utility folder (pn-commons) to allow jpeg image generation starting from an html node.

## List of changes proposed in this pull request
- adds Screenshot module under pn-commons 'utility' folder
- changes Statistics.page adding a global 'export jpeg' button
- removes the download icon from every statistics component
- changes formatDate method in common date.utility to specify a custom separator

## How to test
- Go to sender dashboard and verify the 'download jpeg' button is visible and there's no download icon on statistics components
- Click on the 'download jpeg' button and verify a jpeg screenshot of every statistics component is generated and available to download.
